README.md

```diff
@@ -1,14 +1,34 @@
-# Trike - Programming Language Test
+# Trike
 
-**A simple exercise to expand your programming comprehension**
+## Problem solving
 
-We love programming at Trike. We love to explore what benefits and tradeoffs that other languages have to offer in solving our day to day problems.
+Write a class that applies the following rules (in order of precedence) to any string it is passed:
 
-Although we primarily do a lot of [Ruby](https://www.ruby-lang.org/en/) programming, we also like looking at other languages that help us reason about our code on a daily basis.
+* If a character has the same character to its left and right, it should be replaced with that other
+  character (i.e. AWA becomes AAA) unless the surrounding character is a space
+* Any sequence of six characters should be replaced with a single character, i.e. AAAAAA becomes A
+* Any instance of the exclamation mark (!) character causes the string to be reversed, and the
+  exclamation mark character removed
+
+Commit each step of your development process. When you're done, try running it on the following
+string and see what result you get (hint: you'll know when you've done it right):
+
+```
+!YTIRCO!IQIIQIDEMGMMIM FO YMJMMSM!RA !EGEEJEHT ROEOOSOF PAEJEEBEL TN!AIKIITIG ENVNNMNO ,GQGGCGN!ILEKIZIISIRT A RJRRDROF PETOTTJTS LLZLLEL!AMSXSSMS ENODOOSO
+```
+
+## Expanding your programming comprehension
+
+We love programming at Trike. We love to explore what benefits and tradeoffs that other languages
+have to offer in solving our day to day problems.
+
+Although we primarily do a lot of [Ruby](https://www.ruby-lang.org/en/) programming, we also like
+looking at other languages that help us reason about our code on a daily basis.
```

Comment:
s/ that help us reason about our code on a daily basis//

I don't think it says much.